### DISCUSSION TITLE
Add createUser method on SDK JS

### DISCUSSION THREAD

**Discussion by Silbad** [directus/directus#2823] [Archive]
Hello, why isn't there a createUser() method in SDK JS ?
https://docs.directus.io/guides/js-sdk.html#users
Thanks

**Comment by rijkvanzanten** [comment]
I don't know, but feel free to add it: 

https://github.com/directus/sdk-js/blob/master/src/SDK.ts#L852

**Comment by niklasgrewe** [comment]
@Silbad cool idea 👍 would you like to add it?

**Reply by Silbad** [reply]
Ok I made a pr ;)

**Reply by Silbad** [reply]
I just test with unit test for info ;)

**Reply by niklasgrewe** [reply]
cool thank you so much... 👍 💯

**Reply by rijkvanzanten** [reply]
Published as 6.3.0 🙂
